src/openfarm.yaml

```diff
@@ -0,0 +1,39 @@
+---
+description: >
+  Free and open database for farming and gardening knowledge.
+  Their mission is to develop community and tools for freely sharing plant
+  knowledge locally and globally.
+license: MIT, CC0
```

Comment:
MIT for the code, CC0 for the data. Again, I don't think this matters too much, but if you'd like, we can change the existing projects files to a format similar to:
```yaml
licenses:
  - code: MIT
    for: code
    url: http...
  - code: CC0
    for: data
    url: ...
```

Reply:
I've encountered this before too.
Generally I've referred to the Code's licence as that is what our visitors are most likely to contribute (although documentation is probably not gonna be rare)

The list is probably a good idea, though.

Reply:
Alright, I'll do it today in another PR